### PR TITLE
Use content attachments API

### PR DESCRIPTION
**Pull request by tcbyrd**
Updating this app to use the new content attachments API.

### CODE REVIEW

**Comment by tcbyrd** [comment]
@wilhelmklopp @probot/maintainers Wanted to bump this conversation to get some ideas on how to roll this out. I see a couple of options:

- Remove all issue/pull request unfurls, but add a few domains that can be unfurled explicitly by content reference domains.
- Keep the issue/pull request unfurls in place for non-content reference domains, so some "special" domains will unfurl using content references
- Don't use content references at all and keep this working as it is today.

To do one of the first two, we'd need to decide which domains should be used as `content_references`. Some of the top ideas:
- Twitter
- Stack Overflow
- Reddit
- Spectrum.chat
- GitHub.community (is this allowed?)

Basically any site that uses open graph would work, but these are ones that seem super relevant to developers. I think I'd be ok turning anything else off, since GitHub links are now automatically turned into hover cards, and end up [not rendering properly](https://github.com/probot/unfurl/issues/34) when it's not public.
